src/Amadevus.RecordGenerator.Generators/RecordDescriptor.cs

```diff
@@ -33,6 +36,14 @@ public Entry(SyntaxToken Identifier, TypeSyntax Type, PropertyDeclarationSyntax
 
             public SyntaxToken Identifier { get; }
 
+            public SyntaxToken IdentifierInCamelCase =>
+                Lazy.EnsureInitialized(ref identifierInCamelCase, this, self =>
+                {
+                    var id = (string)self.Identifier.Value;
+                    var id2 = char.ToLowerInvariant(id[0]) + id.Substring(1);
+                    return SyntaxFactory.Identifier(CSharpKeyword.Is(id2) ? "@" + id2 : id2);
+                });
```

Comment:
It's really not a very costly method. I think we can initialize this property directly in a constructor.

Reply:
I'll leave that decision to you and adjust accordingly. Just let me know what you wish to proceed with finally. Also, sorry about the `id2`, which was a horrible slip.

Reply:
> Also, sorry about the `id2`, which was a horrible slip.

FWIW, fixed in dd117f64a1fa3fd666ce33a888364dd5c983cdd2.

Reply:
Let's do initialization in the constructor, the implementation can be moved to a static method.

Reply:
Another reason is that this is used in the generated constructor, so in 99% of scenarios it'll get used anyway.

Reply:
Yeah, except I was thinking of those cases where people would be choosing `None` for `Features` 😜; apologies for the poor taste in or tongue-in-cheek humour.